notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>"and prints out a Conditions Summary table (for all the conditions defined on the checks within the suite):"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>